src/promptflow/promptflow/_cli/_params.py

```diff
@@ -283,6 +283,16 @@ def add_parser_build(parent_parser, entity_name: str):
         action="store_true",
         help=argparse.SUPPRESS,
     )
+    parser.add_argument(
+        "--worker_num",
```

Comment:
let's keep the wording in gunicorn, use --threads and --workers. besides, why they are suppressed?

Reply:
offline synced， keep worker_num and worker_threads  fixed for now (8 & 1)